examples/ArduinoWiFi101/ArduinoWiFi101.ino

```diff
@@ -62,10 +66,6 @@ void loop() {
   }
 }
 
-void messageReceived(String topic, String payload, char * bytes, unsigned int length) {
-  Serial.print("incoming: ");
-  Serial.print(topic);
-  Serial.print(" - ");
-  Serial.print(payload);
-  Serial.println();
+void messageReceived(String topic, String payload) {
```

Comment:
Now that only `String` is provided for the payload, does this prevent handling binary messages . (Fox example, if the payload have 0x00 characters before the end of the data length)